firebase.json

```diff
@@ -7,7 +7,8 @@
   "functions": {
     "predeploy": ["yarn build:functions"],
     "source": "functions",
-    "runtime": "nodejs18"
+    "runtime": "nodejs18",
+    "runtimeConfig": ".runtimeconfig.json"
```

Comment:
Not going to challenge this at all - just haven't worked with `RuntimeConfig` - what do we need this for?